test.rb

```diff
@@ -117,7 +117,6 @@ def around
     Object.send(:remove_const, :User)
     Object.send(:remove_const, :Book)
     Object.send(:remove_const, :Novel)
-    ActiveSupport::Dependencies.clear
```

Comment:
Removing this line causes a test failure for me when running against 6.1.4.4:

```
Error:
DefaultValuePluginTest#test_subclass_find:
ActiveRecord::SubclassNotFound: Invalid single-table inheritance type: Novel is not a subclass of Novel
#... stack trace...

rails test test.rb:372
```

Putting this in place seems to work for both, though:

```ruby
ActiveSupport::Dependencies.clear unless ActiveRecord::VERSION::MAJOR > 6
```

Reply:
@bfad, thanks, I didn’t think about previous versions. :sweat_smile: That looks good. I updated the commit with the change. :ping_pong: